src/main/resources/plugin.yml

```diff
@@ -1,5 +1,5 @@
 name: ${name}
 version: ${version}
 main: com.github.soerxpso.xpspawners.XPSpawners
-
 depend: [CivModCore]
+softdepend: [Citadel, Bastion]
```

Comment:
Are you adding these soft depends for future expansion? At present there is no dependency, I don't believe?